### DISCUSSION TITLE
doctest 2.4.8 and clang-tidy

### DISCUSSION THREAD

**Discussion by acgetchell** [onqtam/doctest#654] [Q&A] [answered]
After moving to doctest, I've noticed that clang-tidy flags quite a few items on doctest. However, it comes up clean on PVS-Studio!

https://www.adamgetchell.org/posts/a-tale-of-two-test-frameworks/

Is there something I should be doing differently?

Thanks for any advice.

**Comment by Saalvage** [comment, marked as answer]
It's an issue we're aware of and we're looking to fix it for the upcoming 2.4.9 release. :)

**Comment by Saalvage** [comment]
To follow up on this: v2.4.9 has been released!
